godtools/App/Flows/LanguageSettings/LanguageSettingsFlow.swift

```diff
@@ -91,7 +91,9 @@ extension LanguageSettingsFlow {
             flowDelegate: self,
             getCurrentAppLanguageUseCase: appDiContainer.feature.appLanguage.domainLayer.getCurrentAppLanguageUseCase(),
             viewDownloadableLanguagesUseCase: appDiContainer.feature.appLanguage.domainLayer.getViewDownloadableLanguagesUseCase(),
-            viewSearchBarUseCase: appDiContainer.domainLayer.getViewSearchBarUseCase()
+            viewSearchBarUseCase: appDiContainer.domainLayer.getViewSearchBarUseCase(),
+            downloadToolLanguageUseCase: appDiContainer.feature.appLanguage.domainLayer.getDownloadToolLanguageUseCase(),
+            removeDownloadedToolLanguageUseCase: appDiContainer.feature.appLanguage.domainLayer.getRemoveDownloadedToolLanguageUseCase()
```

Comment:
I don't think there's a way to remove downloaded languages from the list, however from the language settings those will be able to be removed.

The UseCases do take some time setting up, but it is nice having them because I can see high level what is going on in the ViewModel.

Reply:
Ignore this just saw your comment in DownloadableLanguagesViewModel.